src/content/upgrades/merge/index.md

```diff
@@ -230,23 +190,23 @@ The Ethereum upgrades are all somewhat interrelated. So let’s recap how The Me
 
 ### The Merge and the Beacon Chain {#merge-and-beacon-chain}
 
-The Merge represents the formal adoption of the Beacon Chain as the new consensus layer to the current Mainnet execution layer. Once The Merge happens, validators will be assigned to secure Ethereum Mainnet, and mining on [proof-of-work](/developers/docs/consensus-mechanisms/pow/) will no longer be a valid means of block production.
+The Merge represents the formal adoption of the Beacon Chain as the new consensus layer to the original Mainnet execution layer. Since the Merge, validators are assigned to secure Ethereum Mainnet, and mining on [proof-of-work](/developers/docs/consensus-mechanisms/pow/) is no longer a valid means of block production.
```

Comment:
I haven't been perfect about this, and I don't always love how it ends up looking, but for consistency:
```suggestion
The Merge represents the formal adoption of the Beacon Chain as the new consensus layer to the original Mainnet execution layer. Since The Merge, validators are assigned to secure Ethereum Mainnet, and mining on [proof-of-work](/developers/docs/consensus-mechanisms/pow/) is no longer a valid means of block production.
```